pages/ghg/ghg.md

```diff
@@ -0,0 +1,68 @@
+
+[The GHG Protocol Reporting Standard](https://ghgprotocol.org/sites/default/files/standards/ghg-protocol-revised.pdf) acknowledges that Scope 3 is a difficult and is largely inaccurate to account for. A full life cycle analysis is not required, and the company can focus its accounting effort on GHG generating activities at their discretion. It does recommend some general guidance when assessing Scope 3 emissions:
+
+1. **Describe the value chain.** For transparency, provide a general description of the value chain and associated GHG sources.
+2. **Determine which Scope 3 categories are relevant.** Only some upstream and downstream emissions are relevant to the company, for various reasons:
+  - they are (beleived to be) large relative to Scope 1 and 2 emissions
+  - they contribute to the company's GHG risk exposure and deemed critical by stakeholders
+  - there is opportunity to influence or reduce emissions
+3. **Identify partners along the value chain.** Identify any partners along the value chain that potentially contribute significant GHGs to identify sources, relevant data and calculate emissions.
+4. **Quantify Scope 3 emissions.** It is accepted that Scope 3 emissions may not be accurate due to data availability and reliability. Estimates are acceptable as long as the approach, and methodology is transparent, and that the data used is adequate and reliable.
+
+Inline with this recommended approach, the TCS describes the technology value chain across three categories, identifiying potential sources of technology emissions. The company should carefully consider these categories and sources of emissions that are most relevant to them. Using the [TCS Carbon Estimator tool](/estimator) can help identify which categories and sources may be relatively large and contribute high GHGs. Finally, the TCS provides guidance for quantifying these emissions, with details of appropriate approaches, methodology and sources of reliable data.
+
+The TCS defines three categories that fall within GHG Protocol Scope 3:
+
+{% include categoryLabel.html label="CatU" %}
+
+The upstream value chain; TCS Category U (Upstream), encompasses the embodied carbon of technology including all employee devices, networking equipment and servers. This can make up a significant portion of the organisation's overall GHG emissions. [Product Carbon Footprints](/resources/glossary#product-carbon-footprint-pcf) are generally easily accessible for reporting. However, the upstream impact of purchasing off-the-self and open-source software, is still very difficult to obtain.
```

Comment:
```suggestion
The upstream value chain, TCS Category U (Upstream), encompasses the embodied carbon of technology including all employee devices, networking equipment and servers. This can make up a significant portion of the organisation's overall GHG emissions. [Product Carbon Footprints](/resources/glossary#product-carbon-footprint-pcf) are generally easily accessible for reporting. However, the upstream impact of purchasing off-the-self and open-source software is still very difficult to obtain.
```